helm/templates/ingress.yaml

```diff
@@ -21,6 +21,9 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+{{- if and .Values.ingress.className (semverCompare ">=1.19" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
Does `semverCompare ">=1.19" .Capabilities.KubeVersion.GitVersion` work on EKS? I know they used to have funny versions.

Reply:
Agree with @SimonRupar, let's keep it consistent with the rest of the chart. If there is an issue with EKS versioning, we should open a separate issue for it and fix it in all places.